### PR TITLE
Add gitlab ci, jenkins or discord message plugin

### PR DESCRIPTION
**Pull request by appleboy**
- [Sebd a File or Text Message to Discord (custom define color, username or avatar)](https://github.com/appleboy/telegram-action)
- [Trigger multiple GitLab CI Pipeline](https://github.com/appleboy/gitlab-ci-action)
- [Trigger multiple Jenkins Jobs](https://github.com/appleboy/jenkins-action)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 - [Send email on failed GitHub Checks](https://github.com/cirrus-actions/email)
 - [Report webpack stats to packtracker.io](https://github.com/packtracker/github-action)
 - [Send a Telegram Message](https://github.com/appleboy/telegram-action)
+- [Sebd a File or Text Message to Discord (custom define color, username or avatar)](https://github.com/appleboy/telegram-action)
```
Comment (gep13) [inline README.md]:
```suggestion
- [Send a File or Text Message to Discord (custom define color, username or avatar)](https://github.com/appleboy/telegram-action)
```

**Comment by appleboy** [comment]
@gep13 Done. Thanks.

**Comment by gep13** [comment]
@appleboy thanks for getting that updated!

**Comment by gep13** [comment]
@appleboy your changes have been merged, thanks for your contribution 👍

**Pull request merged**